shared/teams/team/container.js

```diff
@@ -31,13 +31,13 @@ const mapStateToProps = (state: TypedState, {routeProps, routeState}) => {
   return {
     teamname,
     admin: yourOperations.manageMembers,
-    memberCount: state.teams.getIn(['teammembercounts', teamname], 0),
+    memberCount: Constants.getTeamMemberCount(state, teamname),
     _newTeamRequests: state.teams.getIn(['newTeamRequests'], I.List()),
-    numInvites: state.teams.getIn(['teamNameToInvites', teamname], I.Set()).size,
-    numRequests: state.teams.getIn(['teamNameToRequests', teamname], I.Set()).size,
-    numSubteams: state.teams.getIn(['teamNameToSubteams', teamname], I.Set()).size,
+    numInvites: Constants.getTeamInvites(state, teamname).size,
+    numRequests: Constants.getTeamRequests(state, teamname).size,
+    numSubteams: Constants.getTeamSubteams(state, teamname).size,
     loading: anyWaiting(state, Constants.teamWaitingKey(teamname)),
-    resetUserCount: state.teams.getIn(['teamNameToResetUsers', teamname], I.Set()).size,
+    resetUserCount: Constants.getTeamResetUsers(state, teamname),
```

Comment:
Needs a `.size`

Reply:
D'oh! Not sure why flow didn't catch this. I tried adding some types to this file but it didn't seem to help. :(

Reply:
did you run flow coverage?

Reply:
Thanks for reminding me, I'll do that

Reply:
Done, was able to fix the typing (`ownProps` was unknown and was tainting everything else)